server/aesdsocket.c

```diff
@@ -6,7 +6,8 @@
 * The contents obtained are appended in /var/tmp/aesdsocketdata
 * Once the client ends the connection, the full content of the file is returned to it
 * Logs acceptance and closure of connections
-* Accepts new clients until SIGINT or SIGTERM signals are received
+* Accepts new clients redirecting them to threads,
```

Comment:
@jordicrosmompart  you should remove the binary `server/aesdsocket` from your repo and add to .gitignore.